src/com/skypro/Main.java

```diff
@@ -0,0 +1,27 @@
+package com.skypro;
+
+public class Main {
+
+    public static void main(String[] args) {
+	// Задание 1
+        int i = 1;
+        while (i <= 10) {
+            System.out.print(i); i ++;
```

Comment:
можно объединить
```suggestion
            System.out.print(i++);
```
i++ - сначала отдает текущее значение, затем увеличивает i; ++i - наоборот